nodeup/pkg/model/kubelet.go

```diff
@@ -336,6 +343,30 @@ func (b *KubeletBuilder) addStaticUtils(c *fi.ModelBuilderContext) error {
 		}
 	}
 
+	if b.Distribution == distros.DistributionFlatcar {
+		// Flatcar does not ship with conntrack.  Install our own (statically linked) version
+		// TODO: Extract to common function?
+		for _, binary := range []string{"conntrack"} {
```

Comment:
I've checked and I've found out that Flatcar is shipped with `socat` so it is not necessary to install it.

Reply:
That is really good news - the omission of socat was so surprising!